src/piechart/Arc.jsx

```diff
@@ -45,12 +49,9 @@ module.exports = React.createClass({
     // make value text can be formatted
     var formattedValue = props.valueTextFormatter(props.value);
 
-    return (
-      <g className='rd3-piechart-arc' >
-        <path
-          d={arc()}
-          fill={props.fill}
-        />
+    var outerLabels = null;
+    if (this.props.showOuterLabels) {
+      outerLabels = [
```

Comment:
I wasn't able to have to adjacent JSX elements assigned to this var (syntax error), so they're in an array to be passed in below.